src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/FilePathNormalizer.cs

```diff
@@ -12,6 +12,8 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Common
 {
     public class FilePathNormalizer
     {
+        public static readonly FilePathNormalizer Instance = new FilePathNormalizer();
```

Comment:
If we're doing this maybe we should mark the constructor `Obsolete` so that other places use it?

Reply:
Ya not a bad idea. Given the size of a refactoring that'd be I've filed: https://github.com/dotnet/razor-tooling/issues/5918